example/transferKeystoreLocally.js

```diff
@@ -0,0 +1,46 @@
+const evmlib = require('evm-lite-lib');
+
+// Transaction Addresses
+const from = '0xeb0d3811B16f792c119E6395141fa58f8409C597';
+const to = '0xeF5D1019e5B752E993BA17EC3aD2B021872Fb969';
+
+// Transaction Value
+const value = 200;
+
+// Config evm-lite IP
+const nodeIP = '127.0.0.1';
+
+// Data directory object
+const dataDirectory = new evmlib.DataDirectory('/Users/junwei.evmlc');
```

Comment:
Ditto.

Reply:
I have replaced it with simple instructions.